frontend/packages/rhoas-plugin/src/components/service-table/ServiceInstanceTable.css

```
@@ -0,0 +1,3 @@
+.rhoas-plugin--service-table tr>*:empty {
```

Convention is to use each component name part in the css class name.
`rhoas-plugin` is a bit long. Maybe shorten to `rhoas` instead. Furthermore there is inconsistencies in this PR with prefixing your classes. Please make sure they all use `rhoas-plugin` or `rhoas`.

```suggestion
.rhoas-plugin-service-instance-table tr>*:empty {
```
or
```suggestion
.rhoas-service-instance-table tr>*:empty {
```